### ISSUE TITLE
feat(logs): Display Madara logs instead of Substrate logs

### ISSUE THREAD

**Issue by antiyro** [keep-starknet-strange/madara#1484]
# Feature Request

**Describe the Feature Request**

Currently the problem is that we display logs from Substrate on block sync and concensus session:

<img width="846" alt="Capture d’écran 2024-02-29 à 19 10 21" src="https://github.com/keep-starknet-strange/madara/assets/74653697/4738e259-a24c-40e9-9f56-6068b9dd9889">

We should display Madara logs directly instead. The problem is that those logs appear to be displayed within the substrate lib [here](https://github.com/paritytech/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/client/basic-authorship/src/basic_authorship.rs#L572), so we should either override them or do some changes within our Substrate fork.

**Comment by akhercha** [comment]
Hey, happy to take this one!

**Comment by antiyro** [comment]
https://github.com/KasarLabs/deoxys/issues/140

**Comment by antiyro** [comment]
@akhercha yes ser, please check the issue above and lmk if you need any help

**Comment by akhercha** [comment]
Hey guys!
Really sorry but it's pretty hard for me to allocate time for this - I'm un-assigning myself so you can move on with this one!

**Comment by azurwastaken** [comment]
I can take it

**Comment by jbcaron** [comment]
edit link: https://github.com/KasarLabs/deoxys-fork/issues/140

**Comment by DADADAVE80** [comment]
Hello @antiyro , can I work on this if it isn't fixed yet?
